### PR TITLE
Test all the things!

### PR DESCRIPTION
**Pull request by juandopazo**
This pull request adds unit tests for everything in `lib/`.

Appropriate meme:
testallthethings

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by ericf** [comment]
Is this the last step before we have full CI/CD?

**Comment by juandopazo** [comment]
@ericf I want to add tests for `middleware/intl` and see if we can add unit tests for `routes/`. Then we need some functional tests and we'll be ready to go.

**Comment by caridy** [comment]
it is looking good!

**Comment by ericf** [comment]
@juandopazo okay, sounds good. Should this PR be considered WIP then?

**Comment by juandopazo** [comment]
@ericf let me add a test for `middleware/intl` and we can merge this one.

**Comment by juandopazo** [comment]
Ok this is ready to be merged!

File: lib/helpers.js [hunk 1]
```diff
@@ -88,15 +87,25 @@ exports.releaseDownloadUrl = function (name, version) {
     return url.format({
         protocol: 'https:',
         hostname: 'github.com',
-        pathname: path.join(
```
Comment (ericf) [inline lib/helpers.js]: why remove the path.joins?

Comment (juandopazo) [inline lib/helpers.js]: Because `path.join` will use `\` or `/` depending on the platform, but URLs must use `/` all the time.

Comment (ericf) [inline lib/helpers.js]: Okay, good to know, Windows User :)

**Pull request merged**